package.json

```diff
@@ -54,14 +54,14 @@
     "dotenv-cli": "^4.0.0",
     "kcd-scripts": "^11.1.0",
     "npm-run-all": "^4.1.5",
-    "react": "^17.0.1",
-    "react-dom": "^17.0.1",
+    "react": "18.0.0-rc.1",
+    "react-dom": "18.0.0-rc.1",
     "rimraf": "^3.0.2",
     "typescript": "^4.1.2"
   },
   "peerDependencies": {
-    "react": "*",
-    "react-dom": "*"
+    "react": "18.0.0-rc.1",
```

Comment:
Caret range would match `18.0.0-rc.0-next` which is definitely not what we want. 
Will widen to `^18.0.0` once React 18 lands.